docs/clients/mpd.rst

```diff
@@ -59,6 +59,16 @@ Mopidy.
 MPD graphical clients
 =====================
 
+Cantata
+----
+
+`Cantata <https://github.com/CDrummond/cantata>`_ is a graphical MPD client with support for
+Qt4, Qt5, KDE, Linux, MacOSX, Windows, and Ubuntu Touch.
```

Comment:
"macOS" is the current form of branding.

Leave out Ubuntu Touch as that was discontinued a long time ago.